tr/guide/error-handling.md

```diff
@@ -1,24 +1,170 @@
 ---
 layout: page
-title: Express error handling
+title: Express hata işleme
 menu: guide
 lang: tr
+redirect_from: "/guide/error-handling.html"
 ---
-<div id="page-doc" markdown="1">
-# Error handling
+# Hata İşleme
 
-Define error-handling middleware functions in the same way as other middleware functions,
-except error-handling functions have four arguments instead of three:
-`(err, req, res, next)`. For example:
+
+_Error Handling_ refers to how Express catches and processes errors that
```

Comment:
At first, I thought this was an untranslated paragraph, but I now realize that it's just the original English and the Turkish follows.   In general, for localized content, the process we've followed is to REPLACE the en content, not duplicate & translate.  

Unless there is a specific good reason, please remove the en content from the translated page.

Reply:
My mistake, I was supposed to remove the English version once the translation was done. Removed them all, thanks for reviewing.